apps/nextjs-website/src/components/HighlightBox.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+import { HighlightBoxProps } from '@react-components/types';
+import { HighlightBox as HighlightBoxRC } from '@react-components/components';
+import { HighlightBoxSection } from '@/lib/fetch/types/PageSection';
+
+const makeHighlightBoxProps = ({
+  eyelet,
+  title,
+  body,
+  buttonText,
+  buttonHref,
+  imageUrl,
+  sectionID,
+}: HighlightBoxSection): HighlightBoxProps => ({
+  eyelet,
+  title,
+  body,
+  buttonText,
+  buttonHref,
+  imageUrl: imageUrl?.data?.attributes?.url || '',
+  sectionID,
+});
```

Comment:
```suggestion
const makeHighlightBoxProps = ({
  image,
  eyelet,
  link,
  ...rest,
}: HighlightBoxSection): HighlightBoxProps => ({
  imageURL: image.data.attributes.url,
  ...( eyelet && { eyelet }),
  ...( link && { link }),
  ...rest,
});
```
Avoid repeating unchanged props.
Also, no need to check for existence of image as `StrapiImageRequiredSchema` already handles that.
Also, adapt to other suggestions (namely eyelet and link).